src/main/java/com/jagrosh/jmusicbot/commands/dj/ForceRemoveCmd.java

```diff
@@ -0,0 +1,60 @@
+package com.jagrosh.jmusicbot.commands.dj;
+
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.jmusicbot.Bot;
+import com.jagrosh.jmusicbot.audio.AudioHandler;
+import com.jagrosh.jmusicbot.commands.MusicCommand;
+
+
+public class ForceRemoveCmd extends MusicCommand {
```

Comment:
This should extend DJCommand, not MusicCommand

Reply:
Also, use next-line braces instead of inline
```
public class ForceRemoveCmd extends DJCommand
{
```
(and so on and so forth throughout the rest of the file)